index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Hello Microverse</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
```

Comment:
I think it could be a good option to add a header tag and inside of it a nav tag, so your page could be more understandable and easier to navigate through it.